packages/utils/src/script.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import yargs from 'yargs';
+
+import { getLoggerInstance } from './getLoggerInstance';
+
+export const runScriptSync = (script, cleanup) => {
+  const logger = getLoggerInstance();
+  let exitCode = 0;
+
+  try {
+    script();
+  } catch (error) {
+    exitCode = 1;
+    logger.error(error.stack);
+  } finally {
+    if (cleanup) {
+      cleanup();
+    }
+  }
+
+  if (exitCode === 0) {
+    logger.success('Done!');
+  }
+  process.exit(exitCode);
+};
+
+export const runScript = (script, cleanup) => {
+  const logger = getLoggerInstance();
+
+  script()
+    .catch(error => {
+      logger.error(error.stack);
+      process.exit(1);
+    })
+    .then(() => {
+      logger.success('Done!');
+      process.exit(0);
+    })
+    .finally(() => {
+      if (cleanup) {
+        cleanup();
+      }
+    });
+};
+
+/**
+ * @typedef {Object} ScriptConfig
+ * @property {string} [command] Use '*' or '$0' to refer to the default command
```

Comment:
The diff in this file is a bit funny: the changes essentially start here + a copyright header and an import at the top of the file (compare with `runScript.js`)

Reply:
Yep I compared against the original :+1: